### PR TITLE
94 admin info page

### PR DESCRIPTION
**Pull request by Patrickcpoole**
UserProfile portion of the admin info page. 

Need help with the flex-box in the info-container class in the ProfileInfo.css component.

### CODE REVIEW

File: Frontend/src/App.js [hunk 1]
```diff
@@ -11,9 +11,9 @@ import PrivateRoute from './components/Routing/PrivateRoute';
 import { HashRouter, Switch, Route, Redirect } from 'react-router-dom';
 
 function App() {
-  const [authenticated, setAuthenticated] = useState(false);
-  const [token, setToken] = useState(null);
-  const [isAdmin, setIsAdmin] = useState(false);
+  const [authenticated, setAuthenticated] = useState(true);
```
Comment (ethoreson) [inline Frontend/src/App.js]: This actually causes a merge conflict, so before a PR let's set these back to default values for everyone's ease

File: Frontend/src/components/AdminSettings/UserProfileContainer.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.container {
```
Comment (ethoreson) [inline Frontend/src/components/AdminSettings/UserProfileContainer.css]: We're going to change this className to 'userProfileContainer', "container" will spill its designs into all other "container"s

File: Frontend/src/components/AdminSettings/UserProfilesContainer.jsx [hunk 3]
```diff
@@ -1,7 +1,27 @@
...
+  const [data, setData] = useState(UserDummyData);
+  const colors = ['#81B7E3', '#C4C4C4'];
+
+  return (
+    <div className="container">
+      <button>Create Account</button>
+      <div className="profilesContainer">
+        {data.map((userData, index) => (
+          <ProfileInfo
+            color={colors[index % colors.length]}
```
Comment (ethoreson) [inline Frontend/src/components/AdminSettings/UserProfilesContainer.jsx]: <3 good job

File: Frontend/src/components/AdminSettings/UserProfilesContainer.jsx [hunk 4]
```diff
@@ -1,7 +1,27 @@
...
+import UserDummyData from './UserDummyData';
 
 const UserProfilesContainer = () => {
-  return <div>USER PROFILES HERE</div>;
+  const [data, setData] = useState(UserDummyData);
+  const colors = ['#81B7E3', '#C4C4C4'];
+
+  return (
+    <div className="container">
+      <button>Create Account</button>
```
Comment (ethoreson) [inline Frontend/src/components/AdminSettings/UserProfilesContainer.jsx]: Take the "Create Account" button out of this div. The 'container' was wrapping it into a second column. Easy fix

File: Frontend/src/components/AdminSettings/components/components/Dropdown.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+select {
```
Comment (ethoreson) [inline Frontend/src/components/AdminSettings/components/components/Dropdown.css]: we'll consolidate this in another file

**Comment by Patrickcpoole** [comment]
Hey @ethoreson this should be good now but let me know if I missed anything.

**Comment by hannahkamundson** [comment]
@ethoreson @Patrickcpoole  What is the status on this PR?

**Pull request merged**